packages/pilot/src/containers/RecipientDetails/Config/index.js

```diff
@@ -173,9 +206,9 @@ class RecipientDetailConfig extends Component {
           <AnticipationContent
             data={anticipation}
             t={t}
-            onCancel={onCancel}
-            onSave={ainticipationData => onSave(ainticipationData, ANTICIPATION)}
-            onChange={this.onChangeAnticipationHandler}
+            onSave={this.handleSaveAnticipation}
+            onChange={this.handleChangeAnticipationHandler}
+            handleCancel={this.handleCancel}
```

Comment:
apenas para exemplificar, se as sugestões feitas forem aplicadas, aqui nesse ponto ficaria assim:

```suggestion
            onCancel={this.handleCancel}
```

Reply:
Feito!